src/main/java/org/cp/confisual/plantuml/NevisAuthVisitor.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Author : AdNovum Informatik AG
+ */
+
+package org.cp.confisual.plantuml;
+
+import org.cp.confisual.nevisauth.AuthState;
+import org.cp.confisual.nevisauth.Domain;
+import org.cp.confisual.nevisauth.Entry;
+
+public interface NevisAuthVisitor {
+	void visit(AuthState authState);
+	void visit(Entry entry);
+	void visit(Domain domain);
+
+	String getInfo();
```

Comment:
You don't need this method here. A general visitor interface should only define its visit methods.